include/evolution.h

```diff
@@ -61,5 +62,7 @@ enum MegaEvoVariants
 	MEGA_VARIANT_ULTRA_BURST, //Necrozma
 };
 
+#define MB_SHALLOW_WATER 0x17 //For Alolan Raichu
+
```

Comment:
extra space

Reply:
Also from upstream, but could be removed

Reply:
its okay lol